mod-source-record-manager-server/src/test/java/org/folio/services/afterprocessing/AdditionalFieldsUtilTest.java

```diff
@@ -285,4 +288,101 @@ public void shouldNotSortExistingFieldsWhenAddFieldToToMarcRecord() {
     Assert.assertTrue(added);
     Assert.assertEquals(expectedParsedContent, parsedRecord.getContent());
   }
+
+  @Test
```

Comment:
We can make separate tests for each scenario, in future if something change - I think it would be easier for readers to understand and modify particular test and not thinking about changing assert counters for other test cases

Reply:
It makes sense